contracts/cw20-staked-balance-voting/src/tests.rs

```diff
@@ -1218,6 +1218,65 @@ fn test_active_threshold_percentage_invalid() {
             }),
         },
     );
+
+    instantiate_voting(
+        &mut app,
+        voting_id,
+        InstantiateMsg {
+            token_info: crate::msg::TokenInfo::New {
+                code_id: cw20_id,
+                label: "DAO DAO voting".to_string(),
+                name: "DAO DAO".to_string(),
+                symbol: "DAO".to_string(),
+                decimals: 6,
+                initial_balances: vec![Cw20Coin {
+                    address: CREATOR_ADDR.to_string(),
+                    amount: Uint128::from(200u64),
+                }],
+                marketing: None,
+                unstaking_duration: None,
+                staking_code_id: staking_contract_id,
+                initial_dao_balance: Some(Uint128::from(100u64)),
+            },
+            active_threshold: Some(ActiveThreshold::Percentage {
+                percent: Decimal::percent(0),
```

Comment:
Seems like from the language of the test you probably mean `Decimal::percent(101)` here?